DOCS-2784: Update the collect-data quickstarter (#3229)
viamrobotics/docs: docs/get-started/collect-data.md

{{% expand "No computer, camera, or sensor at hand?" %}}
No problem.

> If you have no computer, use [Try Viam](https://app.viam.com/try) to borrow a rover online which is already configured with a camera to test with.

Reviewer: ```suggestion
If you don't have a development machine or other computer that can run `viam-server`, use [Try Viam](https://app.viam.com/try) to borrow a rover online which is already configured with a camera to test with.
```
They still have to have a computer to use try viam

Reviewer: Could also link to those computers here

If you are using a Try Viam rover **start with Step 4** on the **Collect camera images** tab.

> If you have a computer but no physical sensor, you can use the [`viam:viam-sensor:telegrafsensor`](https://app.viam.com/module/viam/viam-telegraf-sensor) model which measures computer performance metrics.

Reviewer: ```suggestion
If you have a computer that can run `viam-server` but no physical sensor, you can use the [`viam:viam-sensor:telegrafsensor`](https://app.viam.com/module/viam/viam-telegraf-sensor) model which measures computer performance metrics.
```

{{% /expand%}}
{{< expand "Step 3: Configure a board" >}}

> Most sensors need to be wired to the pins of SBC such as a [Raspberry Pi](/components/board/pi/).

Reviewer: ```suggestion
Most sensors need to be wired to the pins of a SBC such as a [Raspberry Pi](/components/board/pi/).
```

{{< expand "Step 4: Configure a sensor" >}}

Search the [sensor models](/components/sensor/#available-models) for a model of sensor that is compatible with your sensor hardware.
For example, if you have a Sensirion SHT3x-DIS temperature and humidity sensor, you should use the [`sensirion-sht3xd`](https://docs.viam.com/components/sensor/sensirion-sht3xd/) model of sensor.

> If you don't have physical sensor, you can use the [`viam:viam-sensor:telegrafsensor`](https://app.viam.com/module/viam/viam-telegraf-sensor) model which measures computer performance metrics.

Reviewer: ```suggestion
If you don't have a physical sensor, you can use the [`viam:viam-sensor:telegrafsensor`](https://app.viam.com/module/viam/viam-telegraf-sensor) model which measures computer performance metrics.
```

> If you don't have a physical sensor, you can use the [`viam:viam-sensor:telegrafsensor`](https://app.viam.com/module/viam/viam-telegraf-sensor) model which measures computer performance metrics.

Reviewer: ```suggestion
If you don't have a physical sensor that can be wired to the pins of a SBC, you can use the [`viam:viam-sensor:telegrafsensor`](https://app.viam.com/module/viam/viam-telegraf-sensor) model which measures computer performance metrics.
```
For more clarity
